core/domain/user_domain.py

```diff
@@ -483,6 +492,10 @@ def unmark_banned(self, default_role: str) -> None:
         self.banned = False
         self.roles = [default_role]
 
+    def set_viewed_lesson_info_once_to_true(self) -> None:
+        """Sets user_has_viewed_lesson_info_once to true."""
+        self.user_has_viewed_lesson_info_once = True
```

Comment:
This naming seems a bit weird to me, should it maybe be `mark_user_info_viewed`?

Reply:
Also, the docstring should not only reference variables but should rather explain what this method conceptually does.

Reply:
Ditto in the docstrings below.

Reply:
Updated the docstrings now!